week-2/extended-project/yoda-y-sus-apis/web_api.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "web_api_methods"
+
```

Comment:
Where is the 'csv' gem?

Reply:
The use of gem only use in the file methods with purpose of organize better the code.